### ISSUE TITLE
[FEATURE]: Migrate Pytests from /backend to /training

### ISSUE THREAD

**Issue by dwu359** [DSGT-DLP/Deep-Learning-Playground#907]
### Feature Name

Migrate Pytests from /backend to /training

### Your Name

Daniel Wu

### Description

Write the following tests for /training in /training/tests:

- test_imports.py: Tests if all imports used by the training backend work
- test_loss_function.py: Tests if the loss functions in criterion.py (somewhere in /training/core) work
- test_model_parser.py: Tests if DlModel can parse LayerParams into a model
- test_model.py: Tests if DlModel can correctly build a model
- test_sk_learn_default_dataset_train_test.py: Tests if SklearnDatasetCreator can correctly split a default dataset into train and test datasets given the dataset's name

Use pytest testing framework to implement tests

**Comment by karkir0003** [comment]
ETA: have a PR ready during 9/17/2023

**Comment by karkir0003** [comment]
Prisha was blocked on the import of torch module (probably related to spinning up the wrong backend)

**Comment by karkir0003** [comment]
completed
